src/pages/Pools/Pair/index.tsx

```diff
@@ -98,69 +55,115 @@ export default function Pair({
   if (token0 && (wrappedPair[0] === PairState.NOT_EXISTS || wrappedPair[0] === PairState.INVALID))
     return <Redirect to="/pools" />
   return (
-    <>
+    <SpaceBg>
       <PageWrapper>
-        <SwapPoolTabs active={'pool'} />
-
-        <AutoColumn gap="lg" justify="center">
-          <AutoColumn gap="lg" style={{ width: '100%' }}>
-            <TitleRow style={{ marginTop: '1rem' }} padding={'0'}>
-              <Flex alignItems="center">
-                <Box mr="8px">
-                  <UndecoratedLink to="/pools">
-                    <TYPE.mediumHeader fontWeight="400" fontSize="26px" lineHeight="32px" color="text4">
-                      Pairs
-                    </TYPE.mediumHeader>
-                  </UndecoratedLink>
+        <Box paddingX={3}>
+          <TitleRow>
+            <Flex alignItems="center">
+              <PointableFlex onClick={handleAllClick}>
+                <Box mr="4px">
+                  <DoubleCurrencyLogo
+                    loading={!token0 || !token1}
+                    currency0={token0 || undefined}
+                    currency1={token1 || undefined}
+                    size={20}
+                  />
                 </Box>
-                <Box mr="8px">
-                  <TYPE.mediumHeader fontWeight="400" fontSize="26px" lineHeight="32px" color="text4">
-                    /
-                  </TYPE.mediumHeader>
+                <Box mr="4px">
+                  <Text fontWeight="600" fontSize="16px" lineHeight="20px">
+                    {!token0 || !token1 ? (
+                      <Skeleton width="60px" />
+                    ) : (
+                      `${unwrappedToken(token0)?.symbol}/${unwrappedToken(token1)?.symbol}`
+                    )}
+                  </Text>
+                </Box>
+                <Box>
+                  <ChevronDown size={12} />
                 </Box>
-                <PointableFlex onClick={handleAllClick}>
-                  <Box mr="4px">
-                    <DoubleCurrencyLogo
-                      loading={!token0 || !token1}
-                      currency0={token0 || undefined}
-                      currency1={token1 || undefined}
-                      size={20}
+              </PointableFlex>
+            </Flex>
+            <ButtonRow>
+              <ButtonPurpleDim disabled>{t('trade')}</ButtonPurpleDim>
+            </ButtonRow>
+          </TitleRow>
+          <ContentGrid>
+            <TwoColumnsGrid>
+              <PoolStats loading={wrappedPair[1] === null} pair={wrappedPair[1]} />
+              <DimBlurBgBox padding={'24px'}>
+                <Flex alignItems="center" justifyContent="center" flexDirection={'column'}>
+                  <Box mb={3}>
```

Comment:
I think that this a `justifyContent="space-between"` suits better here (with a full height).  And without the fixed margin.